lib/langs/pt-BR.js

```diff
@@ -4,10 +4,14 @@ module.exports = {
   posts: 'Posts',
   category: 'Categoria',
   categories: 'Categorias',
-  allCategories: 'Todas',
   tag: 'Tag',
   tags: 'Tags',
 
+  // post filter
+  allCategories: 'Todas',
+  search: 'Procurar',
+  searchHint: 'Título e trecho',
```

Comment:
@tgmti 

Would you please take a look at this? 😄 

Leave a comment here and I'll fix it

Reply:
Hello,

i think is better:

```js
  search: 'Pesquisar',
  searchHint: 'Pesquise no título e conteúdo',
```

The "Pesquisar", "Procurar" and "Buscar" words is translate to "Search". 
However, "Pesquisar" it´s more used (ex. Google hint is: "Pesquise no Google")

Reply:
Since you are changing this file, if possible, please change lang.toc to:

```js
// post nav
toc: 'Índice',
```

I was sleepy and did not think straight, I just translated this part verbatim 😄

Reply:
Got it~